File: src/net/eduard/economy/command/EconomyPayCommand.java

package net.eduard.economy.command;

import net.eduard.economy.EduEconomy;
import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;

import net.eduard.api.lib.modules.Mine;
import net.eduard.api.lib.manager.CommandManager;
import net.eduard.api.lib.modules.Extra;
import net.eduard.api.lib.player.FakePlayer;

public class EconomyPayCommand extends CommandManager {

    public EconomyPayCommand() {
        super("pay", "pagar");
        setUsage("/money pagar <jogador> <quantidade>");
        setDescription("Pagar o jogador");

    }

    @Override
    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {

        if (Mine.onlyPlayer(sender)) {
            Player p = (Player) sender;
            if (args.length < 3) {
                sendUsage(sender);
            } else {
                String alvonome = args[1];
                FakePlayer alvoConta = new FakePlayer(alvonome);
                FakePlayer playerConta = new FakePlayer(p);
                Double quantidade = Extra.fromMoneyToDouble(args[2]);
                quantidade = Math.abs(quantidade);


                if (EduEconomy.getInstance().getManager().hasCoins(playerConta, quantidade)) {
                    EduEconomy.getInstance().getManager().addCoins(alvoConta, quantidade);
                    EduEconomy.getInstance().getManager().removeCoins(playerConta, quantidade);
                    sender.sendMessage(EduEconomy.getInstance().message("money-pay").
                            replace("$amount",Extra.formatMoney(quantidade)).
                            replace("$player",alvonome));

                } else {
                    sender.sendMessage(EduEconomy.getInstance().message("money-need"));
                }


            }
        }
        return true;
    }

}

File: src/main/java/net/eduard/economy/command/EconomyPayCommand.kt
package net.eduard.economy.command

import net.eduard.api.lib.manager.CommandManager
import net.eduard.api.lib.modules.FakePlayer
import net.eduard.api.lib.modules.Extra
import net.eduard.economy.EduEconomyPlugin
import org.bukkit.entity.Player
import kotlin.math.abs

class EconomyPayCommand : CommandManager("pay", "pagar") {
    override fun playerCommand(player: Player, args: Array<String>) {
        if (args.size < 2) {
            sendUsage(player)
            return
        }
        val name = args[0]
        val target = FakePlayer(name)
        val playerConta = FakePlayer(player)
        var quantidade = Extra.fromMoneyToDouble(args[1])
        quantidade = abs(quantidade)
        val instance = EduEconomyPlugin.instance
        if (target == playerConta) {
            player.sendMessage(instance.message("cant-pay-self"))
            return
        }
        if (quantidade < 1) {
            player.sendMessage(
                instance.message("money-pay-invalid")
                    .replace("%amount", Extra.formatMoney(quantidade))
            )
            return
        }
        if (instance.manager.hasCoins(playerConta, quantidade, false)) {
            instance.manager.tradeCoins(playerConta, target, quantidade)
            player.sendMessage(
                instance.message("money-pay")
                    .replace("%amount", Extra.formatMoney(quantidade))
                    .replace("%player", name)
            )
            target.sendMessage(
                instance.message("money-payment")
                    .replace("%amount", Extra.formatMoney(quantidade))
                    .replace("%player", player.name)
            )
        } else {
            player.sendMessage(instance.message("money-need"))
        }
    }

    init {
        usage = "/money pagar <jogador> <quantidade>"
        description = "Pagar o jogador"
    }
}